client/src/api_client.js

```diff
@@ -10,38 +11,69 @@ const BASE_URI = process.env.NODE_ENV === 'production'
 const BITS_PATH = '/bits';
 const COMMENTS_PATH = '/comments';
 const CLIENT_SORT_TO_PARAM = { [SORT_DATE]: 'date', [SORT_SCORE]: 'score' };
+// Set this to true in development to use local, fake data instead of making any RPCs.
+const USE_FAKE_CLIENT = false && process.env.NODE_ENV === 'development';
```

Comment:
i think you could also set this using some kind of webpack flag and give the option to add a --use-fake-data flag when starting the environment but maybe that's too much work

Reply:
I think you're exactly right on both counts 😃 well, it's partly laziness, partly... deferred laziness... in that it's annoying to rebuild the app.